Examples/MAX78000/CNN/UNet-highres-demo/project.mk

```diff
@@ -27,7 +29,7 @@ ifeq "$(BOARD)" "EvKit_V1"
 VPATH += TFT/evkit
 endif
 ifeq "$(BOARD)" "FTHR_RevA"
-VPATH += TFT/fthr
+FONTS = LiberationSans16x16
```

Comment:
Was the VPATH line overwritten intentionally? I see other project.mk files in this PR where VPATH += TFT/fthr was left in.

Reply:
Yes, some projects have actual source files place in `TFT/fthr`, whereas others only had the fonts.  I removed the instances where `TFT/fthr` pointed to an empty folder after moving the fonts